src/actions/sources/prettyPrint.js

```diff
@@ -50,18 +74,11 @@ export function createPrettySource(sourceId: string) {
     dispatch(({ type: "ADD_SOURCE", source: prettySource }: Action));
     dispatch(selectSource(prettySource.id));
 
-    const { code, mappings } = await prettyPrint({ source, url });
-    await sourceMaps.applySourceMap(source.id, url, code, mappings);
-
-    // The source map URL service used by other devtools listens to changes to
-    // sources based on their actor IDs, so apply the mapping there too.
-    for (const sourceActor of source.actors) {
-      await sourceMaps.applySourceMap(sourceActor.actor, url, code, mappings);
-    }
+    const { text } = await prettyPrintSource(sourceMaps, prettySource, source);
 
     const loadedPrettySource: JsSource = {
       ...prettySource,
-      text: code,
+      text: text,
       loadedState: "loaded"
```

Comment:
One of the motivations behind the issue for this is that `loadSourceText` already does this `loaded` toggling as part of the `LOAD_SOURCE_TEXT` action for normal sources. I'm hoping that by doing this in loadSourceText that we can remove this part from here entirely.

Reply:
Okay @loganfsmyth, I want to know whether we want to prettyprint all files on their initial loading and remove the dependence on clicking toggle button?